Flutter/en/1.4.11.md

```diff
@@ -0,0 +1,5 @@
+# Success Criterion 1.4.11 - Non-text Contrast
+
+When designing an app, take the contrast ratio of all content into account. The contrast ratio should be at least 3:1.
+
+You can take a screenshot of the app to determine colors and calculate contrast.
```

Comment:
Are there any plugins for Flutter or IDE to check contrast?

Reply:
Not really, as Flutter does not build UI components in the same way as iOS or Android do. I'd reccommend using some contrast calculator found on the web.